### DISCUSSION TITLE
Input type=date default-value

### DISCUSSION THREAD

**Discussion by NeeleWaehnke** [neuefische/questions#411] [Web] [answered]
Hello,
I'm trying to implement an Edit feature, so the user can change the items. I've set the default values for the input fields in edit mode to the existing value (e.g when you only want to change the quantity field, you don't have to enter the name again). However that doesn't work for the date input field, now when one field is changed, the old date is deleted and no date is displayed unless you pick the date again.
I want the date to stay the same (if it's not changed) after saving the changes from editing like it is with the other inputs. 
![Bildschirm­foto 2022-12-16 um 14 09 13](https://user-images.githubusercontent.com/115499721/208105412-288cad87-e668-494d-828e-14f0a610c22d.png)
![Bildschirm­foto 2022-12-16 um 14 08 04](https://user-images.githubusercontent.com/115499721/208105218-017ee086-6d02-4911-b159-7d829b92712b.png)

thanks in advance!

edit: here is my PR: https://github.com/NeeleWaehnke/capstone-project/pull/19

**Comment by MerleMak** [comment]
Hi Neele, please push you current changes and post the link, so I can have a look at your code :)

**Reply by NeeleWaehnke** [reply]
Hi Merle, here is my current PR: https://github.com/NeeleWaehnke/capstone-project/pull/19

**Comment by NeeleWaehnke** [comment, marked as answer]
I found the problem. At the moment I'm using dummy-data from an array to display the Itemcards and the date format in my array is wrong. That's why the date is not saved. When you edit Items that are added via my add-Form it works!
